### ISSUE TITLE
docs: replace `set-output` GitHub Actions command

### ISSUE THREAD

**Issue by LayZeeDK** [ngworker/ngworker#62]
## What is the motivation/use case for changing the behavior?

The `docs` job outputs the following error

```
The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

**Comment by LayZeeDK** [comment]
Our workflow doesn't have direct usage of the `set-output` command. Probably because of `JamesIves/github-pages-deploy-action@4.1.3` which is to be removed in #61.

**Comment by LayZeeDK** [comment]
Fixed by #63.
